### ISSUE TITLE
GET_LIST response is not being handled in index.js

### ISSUE THREAD

**Issue by kodepareek** [kimkha/aor-loopback#1]
Hello. 
First of all thanks a lot of writing this code. Saved me a lot of time. 

However I see that the getList case is not being handled by the code in index.js

cannot really move forward as my first page is a list view of posts. 

Any suggestions?

**Comment by kimkha** [comment]
From my side, I have tons of GET_LIST req on my product. And everything is OK...

So, can you please put your snippet code here?

**Comment by kodepareek** [comment]
Hi Nguen. 

The GET_LIST request seems to be working fine. I have been able to log the server response and it is as expected. However the page does not seem to move ahead after log in. 

I noticed the GET_LIST case for server response in line 108 in index.js. method name convertHTTPResponseToREST

The GET_LIST case is being bypassed and I was wondering if I was to make an implementation of that on my own. 

Again thanks for the quick response.
